### ISSUE TITLE
New language: Armenian

### ISSUE THREAD

**Issue by HamletPetrosyan** [squidfunk/mkdocs-material#4148]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "am",
  "direction": "ltr",
  "clipboard.copy": "Պատճենել",
  "clipboard.copied": "Պատճենված է",
  "edit.link.title": "Խմբագրել այս էջը",
  "footer.previous": "Նախորդը",
  "footer.next": "Հաջորդը",
  "footer.title": "Վերջնագիր",
  "header.title": "Գլխագիր",
  "meta.comments": "Մեկնաբանությունները",
  "meta.source": "Աղբյուր",
  "nav.title": "Տեղորոշում",
  "search.placeholder": "Փնտրել",
  "search.share": "Կիսվել",
  "search.reset": "Ջնջել",
  "search.result.initializer": "Փնտրում",
  "search.result.placeholder": "Մուտքագրեք փնտրելու համար",
  "search.result.none": "Համապատասխանություններ չկան",
  "search.result.one": "1 համապատասխանություն",
  "search.result.other": "# համապատասխանություններ",
  "search.result.more.one": "ևս 1-ը այս էջում",
  "search.result.more.other": "ևս #-ը այս էջում",
  "search.result.term.missing": "Բացակայում է",
  "search.title": "Փնտրում",
  "select.language.title": "Ընտրել լեզուն",
  "select.version.title": "Ընտրել տարբերակը",
  "skip.link.title": "Անցնել պարունակությանը",
  "source.link.title": "Դեպի պահոց",
  "source.file.date.updated": "Վերջին թարմացումը",
  "source.file.date.created": "Ստեղծված է",
  "tabs.title": "Ներդիրներ",
  "toc.title": "Բովանդակություն",
  "top.title": "Վերադառնալ սկիզբ"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks! Added in 63f736bbd. Note that the ISO code for Armenian is `hy`, not `am`.

**Comment by squidfunk** [comment]
Released as part of 8.4.0.
